operator-catalog-initialization-test.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Run operator catalog initialization test on the operator metadata
+  hosts: all
+  become: false
+  gather_facts: false
+
+  vars:
+    operator_work_dir: "/home/jenkins/test-operator"
```

Comment:
`/home/jenkins/agent/test-operator` is used on other places.
e.g. deploy-olm-operator.yml. It would be good to have `/home/jenkins/agent/` on all places.

Reply:
ty,

LGTM